Dockerfile

```
@@ -14,5 +14,7 @@ WORKDIR /pwru
 COPY . .
 RUN make
 RUN chmod a+x /pwru
-ENTRYPOINT ["./pwru"]
-CMD ["./pwru", "-output-tuple"]
+
+FROM ubuntu:22.04 AS base
```

```suggestion
FROM scratch
```
This reduces the final image size to 4MB.

---

Yes this definitely reduced the size of the docker image. This was something new I learnt today, Thanks. :)